docs/document/content/reference/distsql/syntax/rdl/rule-definition/db-discovery/create-db-discovery-type.cn.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "CREATE DB_DISCOVERY TYPE"
+weight = 3
++++
+
+## 描述
+
+`CREATE DB_DISCOVERY TYPE` 语法用于创建数据库发现类型规则
+
+### 语法定义
+
+```sql
+CreateDatabaseDiscoveryType ::=
+  'CREATE' 'DB_DISCOVERY' 'TYPE' databaseDiscoveryTypeDefinition ( ',' databaseDiscoveryTypeDefinition )*
+
+databaseDiscoveryTypeDefinition ::=
+    discoveryTypeName '(' 'TYPE' '(' 'NAME' '=' typeName ( ',' 'PROPERTIES' '(' 'key' '=' 'value' ( ',' 'key' '=' 'value' )* ')' )? ')' ')'
+    
+discoveryTypeName ::=
+  identifier
+
+typeName ::=
+  identifier
```

Comment:
Hi, `identifier` is not up to date, the latest definition is `string`.

Reply:
> Hi, `identifier` is not up to date, the latest definition is `string`.

Okay, I got it. I want to handle this in a uniform way after this PR.